### PR TITLE
Slugification of file names

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Some file names are broken in the dashboard.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/benchmark/plotting.py [hunk 1]
```diff
@@ -28,6 +29,18 @@
...
 
+def _slugify(value: str) -> str:
+    """
+    Normalizes string, converts to lowercase, removes non-alpha characters,
+    and converts spaces to hyphens.
+    """
+    import unicodedata
+    value = unicodedata.normalize('NFKD', value).encode('ascii', 'ignore')  # type: ignore
+    value = unicode(re.sub('[^\w\s-]', '', value).strip().lower())  # type: ignore
+    value = unicode(re.sub('[-\s]+', '-', value))  # type: ignore
```
Comment (jrapin) [inline nevergrad/benchmark/plotting.py:40]: Why does it need type ignore everywhere, what was the error? This is definitely not a good sign.

File: nevergrad/benchmark/plotting.py [hunk 2]
```diff
@@ -28,6 +29,18 @@
 
 # %% Basic tools
 
+def _slugify(value: str) -> str:
+    """
+    Normalizes string, converts to lowercase, removes non-alpha characters,
+    and converts spaces to hyphens.
+    """
+    import unicodedata
```
Comment (jrapin) [inline nevergrad/benchmark/plotting.py:37]:
```suggestion
```

**Review by jrapin — COMMENTED** [review]
Which files are broken, there may be other reasons?

File: nevergrad/benchmark/plotting.py (hunk 1 above)
Comment (teytaud) [reply nevergrad/benchmark/plotting.py]: Fixed.

File: nevergrad/benchmark/plotting.py [hunk 3]
```diff
@@ -28,6 +29,17 @@
...
 
+def _slugify(value: str) -> str:
+    """
+    Normalizes string, converts to lowercase, removes non-alpha characters,
+    and converts spaces to hyphens.
+    """
+    value = str(unicodedata.normalize('NFKD', value).encode('ascii', 'ignore'))
+    value = re.sub('[^\w\s-]', '', value).strip().lower()
+    value = re.sub('[-\s]+', '-', value)
+    # ...
```
Comment (jrapin) [inline nevergrad/benchmark/plotting.py:40]:
```suggestion
```

**Review by jrapin — COMMENTED** [review]
what was the name that broke it? We should add a test with this very case

**Comment by teytaud** [comment]
Not relevant anymore.

**Pull request closed without merging**